parent/pom.xml

```diff
@@ -25,7 +25,7 @@
 
     <properties>
         <astyanax.version>3.8.0-bv8</astyanax.version>
-        <dropwizard-metrics-datadog.version>1.1.1</dropwizard-metrics-datadog.version>
+        <dropwizard-metrics-datadog.version>1.1.13</dropwizard-metrics-datadog.version>
```

Comment:
it's required for` import org.coursera.metrics.datadog.TaggedName;` import otherwise that logic needs to be written manually to support tags (and grouping metrics by table)